slices/libcurl4.yaml

```diff
@@ -0,0 +1,20 @@
+# Easy-to-use client-side URL transfer library (OpenSSL flavour)
+package: libcurl4
+
+slices:
+  libs:
+    essential:
+      - libbrotli1_libs
+      - libc6_libs
+      - libgssapi-krb5-2_libs
+      - libidn2-0_libs
+      - libldap-2.5-0_libs
+      - libpsl5_libs
+      - libssh2-1_libs
+      - libssl3_libs
+      - libnghttp2-14_libs
+      - libzstd1_libs
+      - zlib1g_libs
+      - ca-certificates_data  
```

Comment:
ca-certificates are only `recommended`, so let's pls remove it

Reply:
removed. kindly refresh